internal/common/clients.go

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package common
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/command"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/coredata"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/notifications"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/scheduler"
+)
+
+const (
+	CoreCommandClientName   = "Command"
+	CoreDataClientName      = "CoreData"
+	SchedulerClientName     = "Scheduler"
+	NotificationsClientName = "Notifications"
+)
+
+type EdgeXClients struct {
```

Comment:
Should we put some goDoc comments for the public exposed struct?  Something like 
```
// EdgeXClients is the collection of EdgeX clients useful for Application Service to interact with
```

Reply:
Since this is in Internal we have not been documenting expose APIs

Reply:
ok, nice to know that we have not been doing that :-)